File: backend/internal/app/router/airdrop.go
package router

import (
	"backend/internal/app/api"
	"backend/internal/app/middleware"
	"github.com/gin-gonic/gin"
)

func InitAirdropRouter(Router *gin.RouterGroup) {
	ipfsRouterAuth := Router.Group("airdrop").Use(middleware.JWTAuth())
	{
		ipfsRouterAuth.POST("runAirdrop", api.RunAirdrop)         // 立即触发空投
		ipfsRouterAuth.POST("getAirdropList", api.GetAirdropList) // 获取空投列表
	}
}

File: frontend/src/request/api/airdrop.js
import serviceAxios from "../index";

// ============================ post ============================

    // 获取空投列表
    export const getAirdropList = (data) => {
        return serviceAxios({
            url: `/airdrop/getAirdropList`,
            method: "post",
            data
        })
    }

    // 立即空投
    export const runAirdrop = (data) => {
        return serviceAxios({
            url: `/airdrop/runAirdrop`,
            method: "post",
            data
        })
    }